### PR TITLE
Add examples for false cases in isPrototypeOf doc

### LINKED ISSUE #3831: Issue with "Object.prototype.isPrototypeOf()": No comparative `false` examples
MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/isPrototypeOf

#### What information was incorrect, unhelpful, or incomplete?
Every single example returns `true`, with no contrasting `false` examples

#### Specific section or headline?
All examples in both sections

#### What did you expect to see?
a `false` example for comparison. Users who are trying to learn what `isProtoypeOf` actually is doing don't really gain as much information when every example they see says that `isPrototypeOf` returns `true`, without any contrasting examples of that would return `false` to compare against.

#### Did you test this? If so, how?
N/A

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/javascript/reference/global_objects/object/isprototypeof`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/isPrototypeOf
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/global_objects/object/isprototypeof/index.html
* Last commit: https://github.com/mdn/content/commit/3b80812a1d4f68e57f35fca763f50bf3e5dfe997
* Document last modified: 2021-02-19T19:57:02.000Z

</details>

### PR DESCRIPTION
**Pull request by sideshowbarker**
Fixes https://github.com/mdn/content/issues/3831

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/object/isprototypeof/index.html [hunk 1]
```diff
@@ -75,20 +75,23 @@ <h3 id="Using_isPrototypeOf">Using isPrototypeOf</h3>
...
 
-var baz = new Baz();
-
-console.log(Baz.prototype.isPrototypeOf(baz)); // true
-console.log(Bar.prototype.isPrototypeOf(baz)); // true
-console.log(Foo.prototype.isPrototypeOf(baz)); // true
+const foo = new Foo();
+const bar = new Bar();
+const baz = new Baz();
+
```
Comment (wbamberg) [inline files/en-us/web/javascript/reference/global_objects/object/isprototypeof/index.html:81]:
```suggestion

// prototype chains:
// foo: Foo <- Object
// bar: Bar <- Foo <- Object
// baz: Baz <- Bar <- Foo <- Object
```

This is really a suggestion so up to you. But as I was reading the example and trying to work it out I found myself writing out the prototype chain for each object, and wondered if that might help other people?

Comment (sideshowbarker) [reply files/en-us/web/javascript/reference/global_objects/object/isprototypeof/index.html:81]: > This is really a suggestion so up to you. But as I was reading the example and trying to work it out I found myself writing out the prototype chain for each object, and wondered if that might help other people?

Yes, this is great — thanks. It succinct but adds a lot of clarity

**Review by wbamberg — APPROVED** [review]
👍 this is a really helpful change @sideshowbarker

**Comment by sideshowbarker** [comment]
> 👍 this is a really helpful change @sideshowbarker

All credit to @SampsonCrowley for raising the issue that pointed out the problem

**Comment by SampsonCrowley** [comment]
@sideshowbarker take it all haha all I did was find an issue, you're the one that put the work in

**Pull request merged**